src/magic.cpp

```diff
@@ -175,6 +179,15 @@ static damage_type damage_type_from_string( const std::string &str )
     }
 }
 
+static std::string moves_to_string( const int moves )
```

Comment:
There is already an equal version of this function defined in "magic.cpp". Do we really need another copy if it here?